### PR TITLE
Added a function to split the data into n folds

### PR DESCRIPTION
**Pull request by Saanidhyavats**
This pr solves a part of issue #1068 . Will be creating another pr to solve the other parts.

### CODE REVIEW

**Comment by Saanidhyavats** [comment]
@pdxjohnny I have created a file as discussed in the meeting. Let me know if any change needs to be done.

File: tests/model/test_kfold.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        for i in range(folds):
+            x = i * fold_size
+            fold = list()
+            for j in range(x, x + fold_size):
+                fold.append(data[j])
+            split_data.append(fold)
+        return split_data
+
+
+class TestKfoldTechnique:
```
Comment (pdxjohnny) [inline tests/model/test_kfold.py:25]:
```suggestion
class TestKfoldTechnique(unittest.TestCase):
```

Comment (pdxjohnny) [reply tests/model/test_kfold.py:25]: This will ensure the code gets tested

File: tests/model/test_kfold.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from random import randrange
+
+import pandas as pd
```
Comment (pdxjohnny) [inline tests/model/test_kfold.py:3]: We can't use any third party dependencies within the main `dffml` package

File: tests/model/test_kfold.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class Kfold:
+    def split(
```
Comment (pdxjohnny) [inline tests/model/test_kfold.py:7]: We need to make sure that this works when streaming data. Sources have an `async` `records()` method which yields one record at a time. We have to make sure any method we use doesn't require the whole dataset be in memory at the same time

**Comment by Saanidhyavats** [comment]
@pdxjohnny I have added the test for split method. Currently working on accuracy part.

**Pull request closed without merging**